grpc/interceptors.go

```diff
@@ -0,0 +1,28 @@
+package grpc
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/letsencrypt/boulder/metrics"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type serverInterceptor struct {
+	stats metrics.Statter
+}
+
+func (si *serverInterceptor) intercept(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
+	s := time.Now()
+	si.stats.Inc(fmt.Sprintf("gRPC.%s", info.FullMethod), 1, 1.0)
```

Comment:
If we're going to check for nil, the goal should be to avoid a panic. :-) Maybe return an error in that case?